sale_product_multi_add/wizards/sale_import_products_view.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_import_product_to_sale" model="ir.ui.view">
+        <field name="name">import.products.to.sale.form</field>
+        <field name="model">sale.import.products</field>
+        <field name="arch" type="xml">
+            <form string="Products">
+                <field name="products" />
+                <footer>
+                    <button
+                        name="create_items"
+                        string="Set quantities"
+                        type="object"
+                        class="oe_highlight"
+                    />
+                    or
+                    <button string="Cancel" class="oe_link" special="cancel" />
+                </footer>
+            </form>
+        </field>
+    </record>
+    <record id="view_import_product_to_sale2" model="ir.ui.view">
+        <field name="name">import.products.to.sale.form2</field>
+        <field name="model">sale.import.products</field>
+        <field name="arch" type="xml">
+            <form string="Products">
+                <field name="items">
+                    <tree create="0" editable="bottom">
+                        <field name="product_id" readonly="1" />
+                        <field name="quantity" />
+                    </tree>
+                </field>
+                <footer>
+                <button
+                        name="select_products"
+                        string="Select"
```

Comment:
```suggestion
                        string="Confirm"
```
I believe it is more user-friendly to see Confirm instead of select, as in the end, once you are on that view, you are confirming to add the Sales Order Lines. WDYT?

Reply:
Yes, I agree. Thank you for the suggestion!